opm/simulators/flow/GenericOutputBlackoilModule.hpp

```diff
@@ -65,14 +72,21 @@ class GenericOutputBlackoilModule {
     // Virtual destructor for safer inheritance.
     virtual ~GenericOutputBlackoilModule();
 
-     Scalar* getPRESSURE_ptr(void) {
+    Scalar* getPRESSURE_ptr(void)
+    {
         return (this->fluidPressure_.data()) ;
     };
 
-    int  getPRESSURE_size( void ) {
+    int getPRESSURE_size( void )
+    {
         return (this->fluidPressure_.size()) ;
     };
```

Comment:
If you're doing these kinds of changes here, maybe we could remove the `void` parameter type and the unnecessary semicolon on the functions' closing braces as well?

Reply:
sure. i know it's unrelated but i can't resist doing some cleanups while passing through..